tests/unit_tests/docmaps/provider_test.py

```diff
@@ -204,7 +204,7 @@ def test_should_populate_actions_outputs_with_doi_and_url_manuscript_published_s
                 'type': 'preprint',
                 'doi': DOI_1,
                 'url': f'{DOI_ROOT_URL}{DOI_1}',
-                'published': datetime.fromisoformat('2022-01-01T01:02:03+00:00'),
```

Comment:
Maybe it is better for the field not to be present in this case?
An empty string wouldn't be a valid timestamp. Whereas it being absent signals that we don't have a date.
(If would therefore need to be optional)